### PR TITLE
Do not show the devicetalk when forwarding or sharing.

### PR DESCRIPTION
**Pull request by Hocuri**
Do not show the devicetalk when forwarding or sharing.

### CODE REVIEW

File: src/org/thoughtcrime/securesms/ConversationListFragment.java [hunk 1]
```diff
@@ -361,8 +361,9 @@ private void handleCreateConversation(int chatId) {
     int listflags = 0;
     if (archive) {
       listflags |= DcContext.DC_GCL_ARCHIVED_ONLY;
-    } else if (forwarding) {
-      listflags |= DcContext.DC_GCL_FOR_FORWARDING;
+    } else if (RelayUtil.isRelayingMessageContent(getActivity())) {
+      listflags |= DcContext.DC_GCL_NO_SPECIALS;
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/ConversationListFragment.java:365]: why not just use `DcContext.DC_GCL_FOR_FORWARDING` for both, forwarding and sharing?

**Comment by r10s** [comment]
on forwarding, the device-talk is already removed by DC_GCL_FOR_FORWARDING, for sharing i would suggest to use just the same flag.

File: src/org/thoughtcrime/securesms/ConversationListFragment.java (hunk 1 above)
Comment (Hocuri) [reply src/org/thoughtcrime/securesms/ConversationListFragment.java]: > also, the `forwarding` argument can be probably removed when using the RelayUtil, or?

What do you mean with "The forwarding argument"? The one I already removed?

Comment (Hocuri) [reply src/org/thoughtcrime/securesms/ConversationListFragment.java]: > why not just use DcContext.DC_GCL_FOR_FORWARDING for both, forwarding and sharing?

Yes, makes sense, I changed this.

> also, the `forwarding` argument can be probably removed when using the RelayUtil, or?

What do you mean with "The forwarding argument"? The one I already removed?

File: src/org/thoughtcrime/securesms/ConversationListFragment.java [hunk 2]
```diff
@@ -75,6 +76,7 @@
 
 import static org.thoughtcrime.securesms.util.RelayUtil.REQUEST_RELAY;
 import static org.thoughtcrime.securesms.util.RelayUtil.acquireRelayMessageContent;
+import static org.thoughtcrime.securesms.util.RelayUtil.isForwarding;
```
Comment (cyBerta) [inline src/org/thoughtcrime/securesms/ConversationListFragment.java:79]: at a first glance this looks like an unnecessary import

Comment (Hocuri) [reply src/org/thoughtcrime/securesms/ConversationListFragment.java]: I optimized the imports.

**Pull request merged**